pages/pil/exemptions/routers/modules.js

```diff
@@ -1,42 +1,69 @@
 const { Router } = require('express');
+const { omit, castArray } = require('lodash');
+
 const form = require('../../../common/routers/form');
+const { buildModel } = require('../../../../lib/utils');
+
 const { modules: schema } = require('../schema');
 const { moduleCodes } = require('@asl/constants');
+const { species } = require('@asl/constants');
+const content = require('../content/modules');
+
+const modulesThatRequireSpecies = [
+  'PILA (theory)',
+  'PILA (skills)',
+  'K (theory)',
+  'K (skills)'
+];
```

Comment:
This list is also defined at https://github.com/UKHomeOffice/asl-pages/blob/master/pages/pil/training/routers/modules.js#L7-L12 - it might be worth pulling it out to a shared file (probably still under `pages/pil` though, not all the way into constants)

Reply:
done